src/ui/public/notify/__tests__/notifier.js

```diff
@@ -185,6 +197,60 @@ describe('Notifier', function () {
     });
   });
 
+  describe('#custom', function () {
+    let customNotif;
+
+    beforeEach(() => {
+      customNotif = notifier.custom(customText, customParams);
+    });
+
+    afterEach(() => {
+      customNotif.clear();
+    });
+
+    it('has a custom function to make notifications', function () {
+      expect(notifier.custom).to.be.defined;
+    });
+
+    it('properly merges options', function () {
+      expect(customNotif).to.have.property('title', customParams.title);
+      expect(customNotif).to.have.property('lifetime', customParams.lifetime);
+
+      expect(customParams.title).to.be.a('string');
+      expect(customParams.lifetime).to.be.a('number');
+    });
+
+    it('sets the content', function () {
+      expect(customNotif).to.have.property('content', `${params.location}: ${customText}`);
+      expect(customNotif.content).to.be.a('string');
```

Comment:
`.to.have.property` is better, as states above. Since this one is just checking a string value though, you're right, the type assertion isn't really needed.